gtsfm/scene_optimizer.py

```diff
@@ -33,6 +33,10 @@
 METRICS_PATH = "result_metrics"
 RESULTS_PATH = "results"
 
+# Paths to Save Output in React Folders.
+REACT_METRICS_PATH = "rtf_vis_tool/src/result_metrics"
```

Comment:
same note here as before about relative paths to file, then converted to absolute with .resolve()